src/components/styles/ProjectPreview.styled.ts

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components'
+
+export const PreviewText = styled.div<{ hasAnimation: boolean }>`
+  font-size: 0.9rem;
+  color: ${(props) => props.theme.palette.common.white};
+  @keyframes fadeIn {
+    0% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+  ${(props) => {
+    if (props.hasAnimation) {
+      return `
+      animation-name: fadeIn;
+      animation-delay: 1s;
+      animation-duration: 1.5s;
+      animation-fill-mode: both; 
+      `
+    }
+    return `
+      animation-name: none;
+    `
+  }}
+`
+export const PreviewTitle = styled.div<{ fontSize?: string }>`
+  width: fit-content;
+  justify-content: center;
+  align-items: center;
```

Comment:
do these work without being a flexbox/grid?

Reply:
I removed the width and justify-content as it seems to not be doing much, however preview wrapper has to be display flex.

Reply:
880b6b9e0fc815ddd11f7ba7e8491709bc8311ec